src/jsd/actuator_offline.cc

```diff
@@ -64,6 +64,11 @@ void fastcat::ActuatorOffline::EgdProcess()
   }
 }
 
+void fastcat::ActuatorOffline::EgdClearErrors()
+{
+  // no-op
```

Comment:
Shouldn't we do the following here:

```
jsd_egd_state_.fault_code = JSD_EGD_FAULT_OKAY;
jsd_egd_state_.emcy_error_code = 0;
```

Reply:
no, since there's no way to set a latched JSD-level error in offline mode.